_data/hosting_providers.json

```
@@ -1,4 +1,14 @@
 [
+  {
+    "name": "interhost.co.il",
+    "link": "https://www.interhost.co.il",
+    "category": "full",
+    "tutorial": "https://interhost.co.il/ssl-installation/",
+    "announcement": "",
+    "plan": "https://interhost.co.il/servers-services/cloud-services/virtual-servers/",
```

Per https://github.com/certbot/website#hosting-providers-list, this link isn't going to show up so I think we should just remove it.
```suggestion
    "plan": "",
```